contracts/core/IPermissions.sol

```diff
@@ -17,6 +17,8 @@ interface IPermissions is IAccessControl, IPermissionsRead {
 
     function grantPCVController(address pcvController) external;
 
+    function grantPCVGuard(address pcvGuard) external;
```

Comment:
We aren't going to be upgrading the Core contract, so this addition is not needed.